components.js

```diff
@@ -3,15 +3,52 @@ if (!window.components) {
 }
 
 /**
- * Fill component.
+ * Fill component. Set `slot` to "toolbar" to include in the top toolbar.
```

Comment:
do we offer any other slots right now? Or is `toolbar` just an example? Otherwise, it might make sense to state this out

Reply:
AFAIK it's the only one. 

```suggestion
 * Fill component. Set `slot` to "toolbar" to include in the top toolbar (the only one available now).
```
How about this?

Reply:
As long it's the only one it's okay to have it like the first version